torch/onnx/_internal/exporter.py

```diff
@@ -1023,14 +1083,18 @@ def dynamo_export(
     except Exception as e:
         sarif_report_path = _DEFAULT_FAILED_EXPORT_SARIF_LOG_PATH
         resolved_export_options.diagnostic_context.dump(sarif_report_path)
-        # TODO(bowbao): A summary as well as suggestion for posting the .sarif log in
-        # the issue to be added.
         message = (
             f"Failed to export the model to ONNX. Generating SARIF report at {sarif_report_path}. "
+            f"SARIF is a standard format for the output of static analysis tools. "
+            f"SARIF log can be loaded in VS Code SARIF viewer extension, "
+            f"or SARIF web viewer(https://microsoft.github.io/sarif-web-component/)."
```

Comment:
```suggestion
            "SARIF is a standard format for the output of static analysis tools. "
            "SARIF log can be loaded in VS Code SARIF viewer extension, "
            "or SARIF web viewer(https://microsoft.github.io/sarif-web-component/)."
```
no variable expansion is needed for these lines, so we can save the compute

Reply:
surprised not flagged by linter